src/constants/networks/zkevm.ts

```diff
@@ -0,0 +1,78 @@
+import { ChainId } from '@kyberswap/ks-sdk-core'
+
+import EthereumLogo from 'assets/images/ethereum-logo.png'
+import { KS_SETTING_API } from 'constants/env'
+import { EVMNetworkInfo } from 'constants/networks/type'
+
+const EMPTY = ''
+const EMPTY_ARRAY: any[] = []
+const NOT_SUPPORT = null
+
+const zkEvm: EVMNetworkInfo = {
```

Comment:
I saw they use `pze` for short of this chain

Reply:
kệ họ